src/components/FSHControls.js

```diff
@@ -116,6 +116,7 @@ export default function FSHControls(props) {
   const [fhirVersion, setFhirVersion] = useState('');
   const [dependencies, setDependencies] = useState('');
   const [isGoFSHIndented, setIsGoFSHIndented] = useState(false);
+  const [isGoFSHLineWrap, setIsGoFSHLineWrap] = useState(props.textWrapped);
```

Comment:
Why is this called `isGoFSHLineWrap` and `setIsGoFSHLineWrap`? I think it applies to both sides of the app, so a more appropriate name might be `isEditorLineWrap` and `setEditorLineWrap` -- or even just `isLineWrap` and `setLineWrap`.